resin/logs.py

```diff
@@ -18,104 +21,50 @@ class Logs(object):
     """
 
     def __init__(self):
+        self.base_request = BaseRequest()
         self.config = Config()
         self.device = Device()
+        self.settings = Settings()
 
-    def _init_pubnub(func):
-        @wraps(func)
-        def wrapper(self, *args, **kwargs):
-            if not hasattr(self, 'pubnub'):
-                pubnub_key = self.config.get_all()['pubnub']
-                self.pubnub = Pubnub(
-                    publish_key=pubnub_key['publish_key'],
-                    subscribe_key=pubnub_key['subscribe_key']
-                )
-            return func(self, *args, **kwargs)
-        return wrapper
-
-    @_init_pubnub
-    def subscribe(self, uuid, callback, error):
+    def subscribe(self, uuid, callback):
```

Comment:
There's actually a `count` option here as well.

Reply:
@pimterry, I don't see a count option here https://github.com/resin-io/resin-sdk/blob/master/lib/logs.coffee#L164 or is it added in another branch?

Reply:
Yep, it isn't merged in the SDK yet, it only got mentioned to me last night. SDK PR is here: https://github.com/resin-io/resin-sdk/pull/548